src/admin/shared/components/TopBar/TopBar.tsx

```diff
@@ -9,12 +10,13 @@ import {
 	ToolbarItem,
 	ToolbarLeft,
 } from '@viaa/avo2-components';
+import { RouteConfigComponentProps } from 'react-router-config';
```

Comment:
Deze mag weg, ik denk zelfs dat de `react-router-config` package niet gebruikt wordt in de client.
Dus die dep + `@types/react-router-config` mogen weg.